ui/admin/app/components/form/worker/create-worker-led/index.hbs

```diff
@@ -190,12 +190,9 @@
       </div>
     {{/unless}}
   </div>
-
-  <Rose::LinkButton
-    @route='scopes.scope.workers'
-    @style='secondary'
-    {{on 'click' @refresh}}
-  >
-    {{t 'actions.done'}}
-  </Rose::LinkButton>
+  <Hds::Button
+    @color='secondary'
+    @text={{if @model.isNew (t 'actions.cancel') (t 'actions.done')}}
+    {{on 'click' (fn @cancel @model)}}
```

Comment:
Turns out the model is passed via the `route-action` call, so passing it here is unnecessary.
```suggestion
    {{on 'click' @cancel}}
```

Reply:
ah yes, that is right!